app/routes/oauth-clients/index.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
```

Comment:
**You should not have anything about oauth client, ⚠️  be sure to rebase correctly before merging (by taking by default what's coming from feature/new_ui when it comes to oauth client conflicts !**